hs_core/models.py

```diff
@@ -610,10 +610,13 @@ def update(cls, element_id, **kwargs):
                 except TypeError:
                     raise TypeError("Not a valid date value.")
                 if dt.type == 'created':
-                    raise ValidationError("Resource creation date can't be changed")
+                    #raise ValidationError("Resource creation date can't be changed")
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
                 elif dt.type == 'modified':
-                    dt.start_date = now().isoformat()
-                    dt.save()
+                    # dt.start_date = now().isoformat()
+                    dt.start_date = start_dt
+                    dt.save(force_update=True)
```

Comment:
@selimnairb In resource landing page, "Created:" field should show the start_date you parsed out of the bag, right? The "Last updated:" should show the current date/time when the bag is ingested, right? Do you see "Created:" field not updated with the start_date you parsed out of the bag even though it shows updated value in your debugger? Just want to make sure I understand the issue here.